targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F439xI/TARGET_MODULE_UBLOX_ODIN_W2/sdk/ublox-odin-w2-drivers/OdinWiFiInterface.h

```diff
@@ -1,5 +1,5 @@
 /* ODIN-W2 implementation of WiFiInterface
- * Copyright (c) 2016 u-blox Malm� AB
+ * Copyright (c) 2016 u-blox Malm� AB
```

Comment:
Please revert the change on the copyright

Reply:
I cannot. The file has an unsupported character which automatically gets changed when I modify something in the file and save.

Reply:
ok